README.md

```diff
@@ -20,4 +20,14 @@ This collection requires some additional dependencies to be installed:
 ```shell
 pip install rhoas-sdks --force-reinstall
 ```
-All modules require an 'OFFLINE_TOKEN' environment variable to be set with a valid token. This token is used to authenticate the user. The token is an OpenShift Cluster Manager API Token and can be found here https://console.redhat.com/openshift/token
\ No newline at end of file
+
+## Using collection
+
+All modules require an 'OFFLINE_TOKEN' environment variable to be set with a valid token. This token is used to authenticate the user. The token is an OpenShift Cluster Manager API Token and can be found [here](https://console.redhat.com/openshift/token)
+
+Two further environment variables are required to be set for the collection to work. These environment variables can be placed in a `.env` file. They are:
```

Comment:
Required? They are only for internal needs. No one would use them outside our team.